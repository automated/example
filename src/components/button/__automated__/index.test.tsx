import { runner } from '@automated/automated/lib/dist/main/jest/index';

import automated from '../__automated';

runner(automated);
