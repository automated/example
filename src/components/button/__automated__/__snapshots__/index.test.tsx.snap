// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`/src/components/button test-1 jest-snapshot 1`] = `
<button
  onClick={[Function]}
  style={
    Object {
      "background": "blue",
      "border": "none",
      "borderRadius": 10,
      "color": "white",
      "fontSize": 20,
      "fontWeight": 600,
      "padding": "20px 30px",
    }
  }
  type="button"
>
  Get started!
</button>
`;

exports[`/src/components/button test-2 jest-snapshot 1`] = `
"Snapshot Diff:
- First value
+ Second value

- <Base
+ <button
    onClick={[Function onClick]}
- />
+   style={
+     Object {
+       \\"background\\": \\"blue\\",
+       \\"border\\": \\"none\\",
+       \\"borderRadius\\": 10,
+       \\"color\\": \\"white\\",
+       \\"fontSize\\": 20,
+       \\"fontWeight\\": 600,
+       \\"padding\\": \\"20px 30px\\",
+     }
+   }
+   type=\\"button\\"
+ >
+   Hello automated
+ </button>"
`;

exports[`/src/components/button test-3 jest-snapshot 1`] = `
"Snapshot Diff:
- First value
+ Second value

- <Base
+ <button
    onClick={[Function onClick]}
- />
+   style={
+     Object {
+       \\"background\\": \\"green\\",
+       \\"border\\": \\"none\\",
+       \\"borderRadius\\": 10,
+       \\"color\\": \\"white\\",
+       \\"fontSize\\": 20,
+       \\"fontWeight\\": 600,
+       \\"padding\\": \\"20px 30px\\",
+     }
+   }
+   type=\\"button\\"
+ >
+   Get started!
+ </button>"
`;
